MAUI/NumericEntry/Migration.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfNumericEntry | Syncfusion 
+description: Learn about Migrating from Syncfusion Xamarin NumericTextBox to Syncfusion .NET MAUI NumericEntry control and more.
```

Comment:
Learn about Migrating from Syncfusion Xamarin NumericTextBox and NumericUpDown to Syncfusion .NET MAUI NumericEntry control and more.

Reply:
Addressed the changes.